examples/vmi-sriov.yaml

```diff
@@ -12,9 +23,6 @@ spec:
       - disk:
           bus: virtio
         name: containerdisk
-      - disk:
-          bus: virtio
-        name: cloudinitdisk
```

Comment:
Why have you removed cloud init ?

Reply:
He used the image with all the things pre-prepared already (e.g. the password and relvant drivers).

Reply:
What Edy said and,
Cloudinit in this example does `dhclient eth1` which does nothing in this example, 
I verified with running it with and without cloudinit and check that interface state with `nmcli c s eth1`
it was the same in both cases.
I also dont think we should use dhclient if NetworkManager is enabled

Reply:
How will I - as a user - know which is the password of your pre-baked image ? I'd prefer it to be configured in cloud-init so that I can see it, and know how to connect.

I agree that the `dhclient eth1` instruction is use-less, and that you should remove it.

dhclient + network manager is orthogonal to this discussion.

Long story short, I see value in cloud-init (I know how to connect to that VM) vs your pre-baked img. 

Would keeping your img while leaving cloud init (just configuring the password) be a compromise you're willing to accept ? I think that brings the best of both choices.

Reply:
You right about the password, I will add it to https://github.com/kubevirt/kubevirt/blob/master/containerimages/container-disk-images.md#kubevirt-container-disk-images
If it is documented you ok with dropping it from cloud init as well? if not I will leave it there

Reply:
i also think we should leave it in the yaml, even with a remark (or without since its a simple command),
sometimes its not trivial, nor common practice to find the password on some document,
this document is mostly about creating and using images (but use which is developer oriented not user oriented) which makes it even less trivial to look it there.

Reply:
I'd prefer if it were directly in the example, since it frees the user from going to our docs and find it out.